.github/workflows/run-linting.yml

```diff
@@ -0,0 +1,17 @@
+name: Run Linter
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
maybe we just run this on all pull requests?

Reply:
We could! I don't have a strong opinion either way. My thoughts for not having it on all PRs was that people may have dev branches going into feature branches. Maybe their work is still in progress and they'll get to the linting stuff later on.

Reply:
Ah ok makes sense, I was thinking about our forgetting to add our future develop branch but I guess we can just not forget